storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerConfigTest.java

```diff
@@ -192,4 +194,20 @@ private void assertMaskedSensitiveConfigurations(String configString) {
         Arrays.stream(sensitiveConfigKeys)
                 .forEach(config -> assertTrue(configString.contains(config + "=(redacted)")));
     }
+
+    @Test
+    public void testDefaultMinIsr() {
+        Map<String, Object> props = createValidConfigProps(new HashMap<>(), new HashMap<>(), new HashMap<>());
```

Comment:
Instead of `new HashMap<>`, can it be replaced with `Collections.emptyMap`


```suggestion
        Map<String, Object> props = createValidConfigProps(Collections.emptyMap(), Collections.emptyMap(), Collections.emptyMap());
```